server/sonar-web/src/main/js/apps/issues/sidebar/AssigneeFacet.js

```diff
@@ -114,12 +116,12 @@ export default class AssigneeFacet extends React.PureComponent {
     }
   }
 
-  getStat(assignee: string): ?number {
+  getStat(assignee /*: string */) /*: ?number */ {
     const { stats } = this.props;
     return stats ? stats[assignee] : null;
   }
 
-  renderOption = (option: { avatar: string, label: string }) => {
+  renderOption = (option /*: { avatar: string, label: string } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)